packages/bot-skeleton/src/scratch/hooks/css.js

```diff
@@ -137,14 +137,14 @@ Blockly.Css.CONTENT = [
     '}',
 
     '.blocklyTooltipDiv {',
-    'background-color: #ffffc7;',
-    'border: 1px solid #ddc;',
-    'box-shadow: 4px 4px 20px 1px rgba(0,0,0,.15);',
-    'color: #000;',
-    'display: none;',
-    'font-size: 9pt;',
-    'opacity: 0.9;',
-    'padding: 2px;',
+    'min-height: 34px;',
+    'background-color: #d6dadb;',
+    'border-radius: 4px;',
+    'color: #333333;',
```

Comment:
Please use constants from `packages/shared/src/styles/themes.scss`, I believe it's `--text-prominent`.